src/stats.ts

```diff
@@ -179,6 +194,51 @@ export class StatsClient {
     return this.client.stats('redirect', params as Record<string, any>);
   }
 
+  /**
+   * Adds an amount of item to the cart in the current session. The cart will be automatically
+   * stored in stats if there's any call to registerCheckout. If the item is already in the
+   * cart, the amount is automatically added to the current amount.
+   *
+   * @param params - An options object. See {@link CartItemStatsParams}.
+   * @returns A promise to be fullfilled with the response or rejected
+   * with a `ClientResponseError`.
+   *
+   * @public
+   */
+  public async addToCart(params: CartItemStatsParams): Promise<Response> {
+    validateRequired([params.item_id, params.amount], 'item_id and amount are required');
+    return this.client.stats('add_to_cart', params as Record<string, any>);
+  }
+
+  /**
+   * Removes an amount of item to the cart in the current session. The cart will be automatically
+   * stored in stats if there's any call to registerCheckout. If any of the items' amount drops
+   * to zero or below, it is automatically removed from the cart
```

Comment:
Use the same `@remarks` structure used [here](https://github.com/doofinder/webcomponents/pull/256).